### DISCUSSION TITLE
post/tcpip-1-linux-driver/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#3] [Announcements]
# post/tcpip-1-linux-driver/

本系列 Blog 的初衷是通过自己实现一个 TCP/IP 协议栈的形式来简明易懂地介绍计算机网络相关的知识, 更多细节详见本系列 Ep.0. 相关代码实现开源在 https://github.com/lyc8503/DNet-core.             既然决定开始实现网络协议, 应该从哪里开始呢? 显然, 是物理上的连接到网络开始. 我们的计算机需要通

https://blog.lyc8503.site/post/tcpip-1-linux-driver/

**Comment by tiaotianzhu** [comment]
这个专题还有后续吗哈哈哈哈

**Reply by lyc8503** [reply]
有的兄弟，有的，但由于本系列太流水账没啥意思，后续会换种方式写

请关注 https://blog.lyc8503.net/categories/专题向研究/自己在家开运营商/ 系列喵

**Reply by tiaotianzhu** [reply]
好捏
